server/helpers/oci/oci.go

```diff
@@ -0,0 +1,30 @@
+package oci
+
```

Comment:
Relocate this file under `server/models/designs`?

Reply:
Given a conversation about broader OCI support for Models, too, no its best to centralize OCI encode/decode.